StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethodEnums.swift

```diff
@@ -139,4 +139,68 @@ import Foundation
             return STPLocalizedString("Unknown", "Default missing source type label")
         }
     }
+
+    /// The identifier for the payment method type as it is represented on an intent, e.g. "afterpay_clearpay" for Afterpay
+    @_spi(STP) public var identifier: String {
+        switch self {
+        case .card:
+            return "card"
+        case .alipay:
+            return "alipay"
+        case .grabPay:
+            return "grabpay"
+        case .iDEAL:
+            return "ideal"
+        case .FPX:
+            return "fpx"
+        case .cardPresent:
+            return "card_present"
+        case .SEPADebit:
+            return "sepa_debit"
+        case .AUBECSDebit:
+            return "au_becs_debit"
+        case .bacsDebit:
+            return "bacs_debit"
+        case .giropay:
+            return "giropay"
+        case .przelewy24:
+            return "p24"
+        case .EPS:
+            return "eps"
+        case .bancontact:
+            return "bancontact"
+        case .netBanking:
+            return "netbanking"
+        case .OXXO:
+            return "oxxo"
+        case .sofort:
+            return "sofort"
+        case .UPI:
+            return "upi"
+        case .payPal:
+            return "paypal"
+        case .afterpayClearpay:
+            return "afterpay_clearpay"
+        case .blik:
+            return "blik"
+        case .weChatPay:
+            return "wechat_pay"
+        case .boleto:
+            return "boleto"
+        case .link:
+            return "link"
+        case .klarna:
+            return "klarna"
+        case .linkInstantDebit:
+            return "TODO" // TODO(porter) figure out the correct id here
```

Comment:
@davidme-stripe @jaynewstrom-stripe any idea how link instant debit shows up on an intent? E.g. for Afterpay it's `afterpay_clearpay`. Can't find anything in the codebase.

Reply:
Is this enum value actually used? This might be leftover from an old instant debits implementation

Reply:
@vardges-stripe I see 31 references to it in our codebase, I don't know enough about it to feel comfortable removing it. I may just put something in here like "link_instant_debits", for now this string will only be used for analytics/metrics. Thoughts?

Reply:
@vardges-stripe any FUD with the string I added [here](https://github.com/stripe/stripe-ios/pull/2782/commits/2286b632c4010a6b9fe39e52c1d445056e38d2b0)?

Reply:
nope that's good @porter-stripe please proceed, sorry missed the earlier message.